libsource/src/main/java/com/virtusize/libsource/network/VirtusizeApiTask.kt

```diff
@@ -210,7 +264,15 @@ internal class VirtusizeApiTask(
      * @param responseCode The response code of an API request
      * @param response The response from an API request
      */
```

Comment:
It seems that return has been omitted?

Reply:
Do you mean the comment should have @return? 
Anyway, I did miss documenting the return values, I'll be adding them.

Reply:
@akueisara 
It would be good enough without it.

Reply:
I documented the return values anyway, but I doubt clients will actually look at the code when they encounter a SDK related issue.

Reply:
> but I doubt clients will actually look at the code when they encounter a SDK related issue.
No, they wouldn't see at all 😅 
The purpose would be for internal to be maintained for other people in the future?